routers/api/v1/api.go

```diff
@@ -1250,6 +1250,7 @@ func Routes(ctx gocontext.Context) *web.Route {
 					m.Get("/orgs", org.ListUserOrgs)
 					m.Post("/orgs", bind(api.CreateOrgOption{}), admin.CreateOrg)
 					m.Post("/repos", bind(api.CreateRepoOption{}), admin.CreateRepo)
+					m.Post("/rename", bind(api.RenameUserOption{}), admin.RenameUser)
```

Comment:
This indicates that a rename should be a `POST` on `/users/{username}/rename`, but the `RenameUser` function documents:

```go
	// swagger:operation POST /admin/users/{username} admin adminRenameUser
```

As a result, the API web interface or any API client that's generated from the Swagger definition is going to fail at calling this.

Reply:
@drsybren Thank you for catching that :) I've sent an update.